pricing/src/test/scala/dev/atedeg/mdm/pricing/ActionsTest.scala

```diff
@@ -0,0 +1,122 @@
+package dev.atedeg.mdm.pricing
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+import cats.data.NonEmptyList
+import org.scalatest.GivenWhenThen
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+
+import dev.atedeg.mdm.pricing.utils.*
+import dev.atedeg.mdm.products.Product.*
+import dev.atedeg.mdm.utils.doubleToNumberInOpenClosedRange
+import dev.atedeg.mdm.utils.given
+
+trait ClientMock:
+  private val clientId: ClientID = ClientID(UUID.randomUUID)
+  val client: Client = Client(clientId)
+
+trait PriceListMock:
+  val priceList: PriceList = PriceList(
+    Map(
+      Caciotta(1000) -> 100.euroCents,
+    ),
+  )
```

Comment:
Maybe this could fit in a single line
```suggestion
  val priceList: PriceList = PriceList(Map(Caciotta(1000) -> 100.euroCents))
```